code/modules/spells/spell_types/jaunt/bloodcrawl.dm

```diff
@@ -236,12 +264,18 @@
  * Called when a victim starts to be consumed.
  */
 /datum/action/cooldown/spell/jaunt/bloodcrawl/slaughter_demon/proc/on_victim_start_consume(mob/living/victim, mob/living/jaunter)
+	if(!iscarbon(jaunter))
+		resist_jaunt_damage = TRUE
+		jaunt_damage_timer = null
 	to_chat(jaunter, span_danger("You begin to feast on [victim]... You can not move while you are doing this."))
 
 /**
  * Called when a victim is successfully consumed.
  */
 /datum/action/cooldown/spell/jaunt/bloodcrawl/slaughter_demon/proc/on_victim_consumed(mob/living/victim, mob/living/jaunter)
+	if(!iscarbon(jaunter))
+		resist_jaunt_damage = FALSE
+		jaunt_damage_timer = addtimer(CALLBACK(src, PROC_REF(damage_for_lazy_demon), jaunter), 20 SECONDS)
```

Comment:
Add the `TIMER_STOPPABLE` flag to the `addtimer` call and call `deltimer` when the timer needs to be stopped.